syft/serde.py

```diff
@@ -99,10 +104,26 @@ def serialize(obj: object, compress=True, compress_scheme=LZ4) -> bin:
 
 
 def deserialize(binary: bin, compressed=True, compress_scheme=LZ4) -> object:
+    """ This method can deserialize any object PySyft needs to send or store.
+
+    This is the high level function for deserializing any object or collection
+    of objects which PySyft has sent over the wire or stored. It includes three
+    steps, Decompress, Deserialize, and Detail as described inline below.
+
+    Args:
+        bin (binary): the serialized object to be deserialized
+
```

Comment:
Empty lines between args are not needed.

Reply:
fixed